awx/main/scheduler/task_manager.py

```diff
@@ -118,7 +118,7 @@ def spawn_workflow_graph_jobs(self, workflow_jobs):
                 job = spawn_node.unified_job_template.create_unified_job(**kv)
                 spawn_node.job = job
                 spawn_node.save()
-                logger.info('Spawned %s in %s for node %s', job.log_format, workflow_job.log_format, spawn_node.pk)
+                logger.debug('Spawned %s in %s for node %s', job.log_format, workflow_job.log_format, spawn_node.pk)
```

Comment:
The reason we had logs like these at INFO was because this information wasn't getting tracked anywhere else. Now they get mixed in with the rest of the firehose, which scales with the number of things looked at, instead of the number of things processed.

This would be kind of a troubling loss, but this gives me an idea for a modification to the JobOrigin model. We talked about doing this a better way, but we never got around to it.